src/memory.cpp

```diff
@@ -12,17 +12,31 @@ static unsigned int ulog2(unsigned int numBlocks) {
   return (unsigned int) ceil(log2(std::max(numBlocks, (unsigned int) 2)));
 }
 
-Memory* Memory::create(unsigned int numBlocks, MemEncoding encoding) {
+static unsigned int norm2(unsigned int number) {
```

Comment:
Hm... what is the semantics of this function?
Let's clarify it as a comment.

Reply:
It finds the minimum power of two value greater than equal to input.
I’ll update it in comments